app/controllers/users_controller.rb

```diff
@@ -5,7 +5,9 @@ class UsersController < ApplicationController
   def edit; end
 
   def update
-    if current_user.update_attributes(update_user_params)
+    params = update_user_params
+    user = current_user
```

Comment:
Thanks.  It's totally unnecessary.  I split it out to debug something and accidentally left it in.